index.html

```diff
@@ -28,5 +28,19 @@
   </section>
 </main>
 </body>
-
+<footer>
+  <div class="footer-container">
+    <div class="footer-icons">
+      <div><img src="icons/github.png"></div>
+      <div><img src="icons/insta.png"></div>
+      <div><img src="icons/twitter.png"></div>
+    </div>
+    <div class="footer-madeby">
+      <em>Made with 🤝 by DevDelta on GitHub</em>
+      <div class="footer-license">
+        <em>Released under the MIT license</em>
+      </div>
```

Comment:
We could simplify this a bit and just put the class on the `em` element itself!

```suggestion
      <em class="footer-license">Released under the MIT license</em>
```